### ISSUE TITLE
Installation fails on Ubuntu 20 (Focal Fossa)

### ISSUE THREAD

**Issue by thbar** [ANXS/postgresql#489]
I am in the process of migration a few machines to Ubuntu 20.

I get the following error:

```
TASK [ANXS.postgresql : PostgreSQL | Extensions | Make sure the postgis extensions are installed | Debian] ***********************
fatal: [vagrant-xyz]: FAILED! => changed=false 
  msg: No package matching 'libgeos-c1' is available
```

This package is not available, and I believe that like in https://github.com/ANXS/postgresql/pull/435, the correct one is `libgeos-c1v5`:

```
$ sudo apt-cache search -n libgeos
libgeos++-dev - Geometry engine for GIS - C++ development files
libgeos-3.8.0 - Geometry engine for Geographic Information Systems - C++ Library
libgeos-c1v5 - Geometry engine for Geographic Information Systems - C Library
libgeos-dev - Geometry engine for GIS - Development files
libgeos-doc - Documentation for the GEOS GIS geometry engine library
```

**Comment by mikysal78** [comment]
> I am in the process of migration a few machines to Ubuntu 20.
> 
> I get the following error:
> 
> ```
> TASK [ANXS.postgresql : PostgreSQL | Extensions | Make sure the postgis extensions are installed | Debian] ***********************
> fatal: [vagrant-xyz]: FAILED! => changed=false 
>   msg: No package matching 'libgeos-c1' is available
> ```
> 
> This package is not available, and I believe that like in #435, the correct one is `libgeos-c1v5`:
> 
> ```
> $ sudo apt-cache search -n libgeos
> libgeos++-dev - Geometry engine for GIS - C++ development files
> libgeos-3.8.0 - Geometry engine for Geographic Information Systems - C++ Library
> libgeos-c1v5 - Geometry engine for Geographic Information Systems - C Library
> libgeos-dev - Geometry engine for GIS - Development files
> libgeos-doc - Documentation for the GEOS GIS geometry engine library
> ```

idem problem with debian buster and skip with vars into playbook:
```yml
    postgresql_ext_postgis_deps:
      - libgeos-c1v5
      - "postgresql-{{ postgresql_version }}-postgis-{{ postgresql_ext_postgis_version }}"
      - "postgresql-{{ postgresql_version }}-postgis-2.5-scripts"
```

**Comment by MrMegaNova** [comment]
I close this issue because wa can't reproduce, thus now we've got mulecule test with debian and this role is now extensively tested on many debian versions
